src/client/datascience/variablesView/variableView.ts

```diff
@@ -21,26 +20,23 @@ import { sendTelemetryEvent } from '../../telemetry';
 import { Telemetry } from '../constants';
 import { IDataViewerFactory } from '../data-viewing/types';
 import { DataViewerChecker } from '../interactive-common/dataViewerChecker';
-import { KernelState, KernelStateEventArgs } from '../notebookExtensibility';
 import {
     ICodeCssGenerator,
     IJupyterVariableDataProviderFactory,
     IJupyterVariables,
     IJupyterVariablesRequest,
-    INotebookEditor,
-    INotebookEditorProvider,
-    INotebookExtensibility,
+    INotebook,
```

Comment:
Goal was for the VariableView to not need awareness of anything Editor or Native Notebook specific.